auth_totp/models/res_users.py

```diff
@@ -50,53 +56,38 @@ def _check_enabled_with_authenticator(self):
 
     @api.model
     def check_credentials(self, password):
-        try:
+        """Add MFA logic to core authentication process
```

Comment:
Should end your docblocks with periods ([PEP-0257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) Single line docstring note 4). 

The bullet points should also have punctuation, because they are sentences that complete the introductory stem (`Overview:`)